app/lib/csv/buyers_exporter.rb

```diff
@@ -16,6 +16,7 @@ def generate
   end
 
   def values_for_account(account)
+    user = (account.admin_user || User.new).decorate
```

Comment:
Hmm. If we ever implement some defaults to attributes of `User`, this can lead to some quite unexpected CSV outputs. I think better ensuring `account.admin_user` is present or the user columns are `nil`. WDYT?

Reply:
Makes sense 👍